src/Symfony/Component/Lock/Store/MemcachedStore.php

```diff
@@ -74,7 +74,7 @@ public function save(Key $key)
      */
     public function waitAndSave(Key $key)
     {
-        @trigger_error(sprintf('%s has been deprecated since Symfony 4.4 and will be removed in Symfony 5.0.', __METHOD__));
+        @trigger_error(sprintf('%s() is deprecated since Symfony 4.4 and will be removed in Symfony 5.0.', __METHOD__));
```

Comment:
I have updated all of this in a PR already ;)

Reply:
I must have missed that one. :) Do I am gonna rebase here afterwards.